### PR TITLE
Feature/icon page

### PR DESCRIPTION
**Pull request by ChanJit**
** Please provide as much detail as possible, but feel free to remove irrelevant sections **

The reason for us to have a icon page is to easy us find icon and avoid adding duplicate icon into seekasia style guide. 

BREAKING CHANGE:

every icon folder been added a file `.iconSketch.js`, icon page will only pick up this file to do rendering

RFC URL:

none

MIGRATION GUIDE:

none

PRINT SCREEN:
localhost_3000_icons 1

SIDE NOTE:
do found out some icons are actually same. such as 
1) BookmarkIcon = IconBookmark
2) CompanyIcon = IconCompany
3) EducationIcon = IconEducation
4) EmployerIcon = IconEmployer
5) HomeIcon = IconHome
6) CheckMarkIcon = TickIcon

and some of them is a bit similar to each other such as 
1) ClearIcon = IconNegative
2) IconPositive = TickCircleIcon

### CODE REVIEW

File: docs/src/components/Icons/Icons.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    <div>
+      <PageBlock>
+        <Section header>
+          <Text screaming>Icons</Text>
+        </Section>
+        <Card transparent style={{ maxWidth: 720 }}>
+          <Section>
+            <Paragraph>
+              <Text>
+                This page is list out all the icons in seekasia style guide.
```
Comment (mengtzu) [inline docs/src/components/Icons/Icons.js]: Literally would be `This page lists out all icons in the SEEK Asia style guide`

You probably want something like `These are the icons components available for use in the SEEK Asia style guide`

We should make sure icons not represented in the brand portal don't have the sketch/demo files needed to show up here.

**Comment by mengtzu** [comment]
Huge thanks for doing this, should make icons much less of a headache.

Might be worth having a guide to using them in code?  Also an explanation that the variants you've marked with `-->` represent props?

**Comment by patrickliu-sa** [comment]
Do we need a first round clean up on the existing icons before merge? If yes I can spare some time to fix them first

**Comment by mengtzu** [comment]
@patrickliu-sa yeah I think so, a quick cleanup is worth doing.  Anything that looks like it might take a while should probably be a new branch after this is merged.

**Comment by ChanJit** [comment]
hey @mengtzu for the duplication icons should remove or remain ?

**Comment by mengtzu** [comment]
Remove the non SEEK Asia ones from the demo page, you can leave the components themselves there.

**Comment by patrickliu-sa** [comment]
Let me do a round of normalization with below specs:

- icon default canvas dimension set to 24 pixels width and 24 pixels height.
- icon default color will fill with currentColor (e.g. icon in `<p style="color:cyan;"></p>` block will pickup the cyan color unless specified manually). If no font color defined, color "Black" will be used as fallback.
- all icons use closed paths, no stroke is set unless specified manually.
- all icon raw svg files will optimized via [SVGO](https://jakearchibald.github.io/svgomg/).

**Comment by mengtzu** [comment]
Awesome 👍 Make sure you coordinate on slack, @patrickliu-sa @ChanJit but great to see it coming together

**Pull request merged**